### ISSUE TITLE
[Emacs 26.3 + Prelude] M-x package-install can't find dracula theme

### ISSUE THREAD

**Issue by ksielewi** [bbatsov/prelude#1253]
Hi Guys,

I have installed the latest Emacs 26.3 on my Ubuntu 18.04.4. I tried to install the Dracula theme following https://draculatheme.com/emacs [M-x package-install <RET> dracula-theme]. The package is not present on the list.

Could I please ask you for advice on how I can fix that?

Best regards and stay safe,
Krzysztof

**Comment by TheZano** [comment]
try `M-x package-refresh-contents` first

**Comment by ksielewi** [comment]
Hi,

I tried refreshing package contents and my Emacs still does not see the Dracula theme.

Best regards,
Krzysztof

**Comment by TheZano** [comment]
are you using this repo for your config or you just started with vanilla emacs? the theme you've mentioned is on Melpa, you might want to add this line `(add-to-list 'package-archives
             '("melpa-stable" . "https://stable.melpa.org/packages/") t)`

**Comment by ksielewi** [comment]
Hi,

I installed the latest Emacs via Ubuntu's packet manager (Emacs 26.3), then I installed Prelude.

1. When I do M-x package-refresh-contents, I see an information that Emacs contacts Elpa and then Melpa, and finally it says Package refresh done.

2. I see that Melpa is already added in prelude-packages.el fine

What I don't understand is as Melpa seems to be added and Emacs contacts Melpa while doing package-refresh-contents why doesn't it see Dracula on the list of available packages?

Thanks
